pages.fr/common/git-check-ignore.md

```diff
@@ -5,20 +5,20 @@
 
 - Vérifie qu'un fichier ou répertoire est ignoré :
 
-`git check-ignore {{chemin/vers/fichier_ou_repertoire}}`
+`git check-ignore {{chemin/vers/fichier_ou_répertoire}}`
```

Comment:
I think we should define if we will allow or deny accents or non-English chars in this variables to enforce to the read that this is a variable. As a native Spanish speaker I don't like them.

Reply:
@navarroaxel I agree with you, it should be decided and after that we could follow the rule (this should also be added somewhere in the guidelines for translation at the end).
For me, it does not matter if we allow or deny the accents in the variables.
I've put accents here because I was acting as the automatic speller checker I saw which made automatic correction of non accentuated words.
It seems that your choice of denying them could be seen as reasonable as I'm not sure a variable's name can use these kind of accents in language.
Can other people give their feedback on these?
If we choose not to use accents in variable's name, I will happily replace them.

Reply:
I'm not sure if we can generalize this for all languages. How about we open an issue and wait some time until we get opinions from various language's native speakers? In the end maybe we can do it for some languages, but not for others.

Reply:
Personally, I see no reason why they should be removed.  Commands work just as well with accented arguments as with plain arguments:
```console
$ touch àbçdēfghïjkłmñœpqrßtùvwxÿż
$ echo 'Hello' > àbçdēfghïjkłmñœpqrßtùvwxÿż
$ ls | grep àbçdēfghïjkłmñœpqrßtùvwxÿż
àbçdēfghïjkłmñœpqrßtùvwxÿż
$ cat ./àbçdēfghïjkłmñœpqrßtùvwxÿż
Hello
$ rm $(pwd)/àbçdēfghïjkłmñœpqrßtùvwxÿż
$ ls | grep àbçdēfghïjkłmñœpqrßtùvwxÿż
$
```

Reply:
I think it's mostly stylistic. In German, we only have three accent letters (üöä) and it's kind of weird to see them in code or commands.

Reply:
But changing the word (as removing the accents is, is it not?) seems wrong.  We might as well leave the entire variables English then 🤔

Reply:
I can't speak about the correctness of removing them, but I can understand arguments for it. I think we should discuss that in a dedicated issue.

Reply:
I'll open an issue now.

Reply:
#5924

Reply:
I would suggest keeping the accents unless there's a clear stylistic reason in that language.